src/abi_llvm.cpp

```diff
@@ -0,0 +1,38 @@
+//===-- abi_llvm.cpp - LLVM Target ABI description --------------*- C++ -*-===//
+//
+//                         LDC – the LLVM D compiler
+//
+// This file is distributed under the BSD-style LDC license. See the LICENSE
```

Comment:
That LICENSE file is not shipped with Julia AFAICT. Also, "BSD-style" is not very specific. If that's not the pure BSD (and which version of the BSD license?), probably better copy the license text in the header.